AirCasting/Utils/TimeConversion.swift

```diff
@@ -0,0 +1,22 @@
+// Created by Lunar on 09/09/2021.
+//
+
+import Foundation
+
+func is24Hour() -> Bool {
+    let dateFormat = DateFormatter.dateFormat(fromTemplate: "j", options: 0, locale: Locale.current)!
```

Comment:
initialization time of a `DateFormatter` is a really time consuming task, we probably should be defining this in a static (if you make this an object it will be possible)